ci-operator/step-registry/storage/create/csi-aws-efs/sts-operator-role/storage-create-csi-aws-efs-sts-operator-role-commands.sh

```diff
@@ -0,0 +1,63 @@
+#!/bin/bash
+set -o errexit
+set -o nounset
+set -o pipefail
+
+export AWS_SHARED_CREDENTIALS_FILE="${CLUSTER_PROFILE_DIR}/.awscred"
+
+# For disconnected or otherwise unreachable environments, we want to
+# have steps use an HTTP(S) proxy to reach the API server. This proxy
+# configuration file should export HTTP_PROXY, HTTPS_PROXY, and NO_PROXY
+# environment variables, as well as their lowercase equivalents (note
+# that libcurl doesn't recognize the uppercase variables).
+if test -f "${SHARED_DIR}/proxy-conf.sh"
+then
+	# shellcheck disable=SC1090
+	source "${SHARED_DIR}/proxy-conf.sh"
+fi
+
+AWS_ACCOUNT_ID=$(aws sts get-caller-identity --query "Account" --output text)
+CLUSTER_ID=$(oc get infrastructure cluster -ojsonpath='{.status.infrastructureName}')
+OIDC_PROVIDER=$(oc get authentication cluster -ojsonpath='{.spec.serviceAccountIssuer}' | sed -e "s/^https:\/\///")
+NAMESPACE="openshift-cluster-csi-drivers"
+POLICY_ARN_STRINGS="arn:aws:iam::aws:policy/AmazonS3FullAccess
+arn:aws:iam::aws:policy/AmazonElasticFileSystemFullAccess"
```

Comment:
Our [docs](https://docs.openshift.com/rosa/storage/container_storage_interface/persistent-storage-csi-aws-efs.html#efs-sts_persistent-storage-csi-aws-efs) show very different permissions, should we test with those?

Reply:
Good catch! Better to update with origin doc shows, I use the aws preset policies to avoid create new ones. Let me update it consistently with our official doc.